.github/actions/build/action.yml

```diff
@@ -61,6 +61,7 @@ runs:
         NEXT_PUBLIC_TRANSAK_APP_URL: 'https://global.transak.com'
         NEXT_PUBLIC_TRANSAK_API_URL: 'https://api.transak.com'
         NEXT_PUBLIC_TRANSAK_API_KEY: '799087ea-c2be-44b8-a0fe-56d96b37ee3c'
+        NEXT_PUBLIC_TURNSTILE_SITE_KEY: 0x4AAAAAAABe-kMUW_1JUfT7
```

Comment:
Would removing this and using it as a secret, instead, add any layer of protection over the faucet than already exists? Are we okay publishing this or would we benefit from obscuring it and help keep the faucets up and running